File: Java/백준/Bronze/10818. 최소， 최대/최소， 최대.java
import java.util.*;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int min = 1000001, max = -1000001;
        int num = sc.nextInt();
        int[] arr = new int[num];

        for(int i = 0; i < num; i++){
            arr[i] = sc.nextInt();
        }

        for(int i = 0; i < num; i++){
            if(arr[i] < min){
                min = arr[i];
            }
            if(arr[i] > max){
                max = arr[i];
            }
        }

        System.out.println(min + " " + max);
    }
}

File: C++17/백준/Bronze/10818. 최소， 최대/최소， 최대.cc
#include <iostream>
using namespace std;

int main(void) {
	int* arr = new int[10000000];
	int n = 0;
	cin >> n;
	int num, max = 0, min = 0;
	for (int i = 0; i < n; i++) {
		cin >> num;
		arr[i] = num;
	}
	max = arr[0];
	min = arr[0];
	for (int i = 0; i < n; i++) {
		max = max > arr[i] ? max : arr[i];
		min = min < arr[i] ? min : arr[i];
	}
	cout << min << endl;
	cout << max << endl;
	delete[] arr;
	return 0;
}